CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ parent: README
 nav_order: 1
 ---# Changelog
 
+## [2.0.9] - 2020-10-22
+- Add support for M3 DB and M3 Aggregator
```

Comment:
It might be good to say that this is beta functionality or something like that. There's e.g. currently a change happening to the components being reported for M3 so one shouldn't rely on things being stable yet.

Reply:
Added more about the beta, please have a look.